src/components/CreateContact/CreateContact.js

```diff
@@ -16,7 +17,6 @@ export default function CreateContact({ onSubmit }) {
               id="firstName"
```

Comment:
If you're using the id in your input element, you can use it in your LabelStyled with `htmlFor="firstName"` for when the user is clicking the label, the input for this label is focused.